src/algorithms/all_paths.c

```diff
@@ -10,19 +10,21 @@
 
 // Make sure context levels array have atleast 'level' entries,
 // Append given 'node' to given 'level' array.
-static void _AllPathsCtx_AddNodeToLevel(AllPathsCtx *ctx, uint level, Node *node) {
+static void __AllPathsCtx_AddConnectionToLevel(AllPathsCtx *ctx, uint level, Node *node,
```

Comment:
One underscore is sufficient - `_AllPathsCtx_AddConnectionToLevel`.

Reply:
done